espaloma/mm/nonbonded.py

```diff
@@ -71,6 +71,25 @@ def lorentz_berthelot(g, suffix=""):
 
     return g
 
+def get_q(g, suffix=""):
```

Comment:
Every method should have a docstring documenting what the method does and what the parameters and return values are (with types and sizes noted).

Could we use more descriptive function names like `get_charges` instead of the inscrutable `get_q`?

Reply:
Since this method appears to modify the graph passed in and _then_ also return the modified graph, it's all the more important to document its effects.